svc/IdentityService.js

```diff
@@ -73,6 +73,11 @@ export class IdentityService {
      * and refreshes the application to present a login panel.
      */
     async logoutAsync() {
+        try {
```

Comment:
I find the mix of async/exception styles in this method a little bit confusing.   Also, why should't an app exception during logout just be surfaced to the user in the same way a framework exception would be?  Seems like we add the call to logoutAsync to the top of the promise chain.